### DISCUSSION TITLE
Welcome to live-share-sdk Discussions!

### DISCUSSION THREAD

**Discussion by halbondmsft** [microsoft/live-share-sdk#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of the Live Share community during our public preview. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by corinagum** [comment]
Hi all, my name is Corina and I'm a part of the Live Share developer team! I hope you're as excited about the announcement of Live Share as we are!


------

Here are some resources from Microsoft Build 2022 where you can learn more!

## [Satya Nadella's Build 2022 keynote](https://www.youtube.com/watch?v=971YIvosuUk&t=9s)

[<img width="885" alt="Satya Nadella 2022 keynote image" src="https://user-images.githubusercontent.com/14900841/170113585-01cb726b-3707-494d-84ed-b45187e2b36e.png">](https://www.youtube.com/watch?v=971YIvosuUk&t=9s)

## [Microsoft Developer Blog: Introducing Live Share](https://devblogs.microsoft.com/microsoft365dev/introducing-live-share-interactive-app-experiences-in-microsoft-teams-meetings/)
[<img width="1132" alt="Introducing Live Share blog image" src="https://user-images.githubusercontent.com/14900841/170113801-124b861c-01ce-46b8-a7bc-be0b46aac151.png">](https://devblogs.microsoft.com/microsoft365dev/introducing-live-share-interactive-app-experiences-in-microsoft-teams-meetings/)

**Comment by nquyhuynhwork** [comment]
Hi @corinagum ,
I'm following the sample at link https://github.com/microsoft/live-share-sdk/tree/main/samples/02.react-video. Sample app is added to Tab but "Watch together" button is not working! Any ideals?
<img width="1059" alt="Screen Shot 2022-07-10 at 13 32 07" src="https://user-images.githubusercontent.com/25605518/178134469-3de7ae4c-4cdf-4b33-8db2-c3ac728f7a88.png">

**Reply by ryanbliss** [reply]
Hi @nquyhuynhwork, the watch together button will only work from the meeting side panel. Join the meeting and the "Contoso" tab will appear alongside "Chat" and "People". Then try again. 

If that doesn't work, make sure you are in Developer Preview. Let us know if you are still having trouble!
